sdk/spring/CHANGELOG.md

```diff
@@ -64,6 +96,19 @@ This section includes changes in `spring-cloud-azure-stream-binder-eventhubs` mo
 
 #### Breaking Changes
 - Change the type of the binding producer property of `send-timeout` from `long` to `Duration` [#26625](https://github.com/Azure/azure-sdk-for-java/pull/26625).
+- Change the message header prefix from `azure_eventhub` to `azure_eventhubs_` [#27746](https://github.com/Azure/azure-sdk-for-java/pull/27746).
+
+### Spring Cloud Azure Starter Integration Event Hubs
+This section includes changes in the `spring-cloud-azure-starter-integration-eventhubs` module.
+
+#### Breaking Changes
+- Change the message header prefix from `azure_eventhub` to `azure_eventhubs_` [#27746](https://github.com/Azure/azure-sdk-for-java/pull/27746).
```

Comment:
I think we don't need to list the changes for each starter, if there're changes for a starter, such changes should be listed in the auto-configuration module since the starter is just a pom file and all codes are in the auto-configuration module.